paper/paper.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'OceanBioME.jl: A flexible environment for modelling the coupled interactions between ocean biogeochemistry and physics'
+tags:
+  - julia
+  - biogeochemistry
+  - climate
+  - ocean
+  - carbon
+authors:
+  - name: Jago Strong-Wright
+    orcid: 0000-0002-7174-5283
+    corresponding: true
+    affiliation: "1, 2"
+  - name: Si Chen
+    affiliation: "1, 2"
+  - name: Navid C Constantinou
+    affiliation: 4, 5
+  - name: Simone Silvestri
+    affiliation: 3
+  - name: Gregory LeClaire Wagner
+    orcid: 0000-0001-5317-2445
+    affiliation: 3
+  - name: John R Taylor
+    affiliation: "1, 2"
+affiliations:
+ - name: Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Cambridge, United Kingdom
+   index: 1
+ - name: Centre for Climate Repair, Cambridge, United Kingdom
+   index: 2
+ - name: Massachusetts Institute of Technology
+   index: 3
+ - name: Australian National University
+   index: 4
+ - name: Australian Research Council Centre for Climate Extremes
+   index: 5
+date: 15 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+``OceanBioME.jl`` is a flexible modelling environment written in Julia [@julia] for modelling the coupled interactions between ocean biogeochemistry, carbonate chemistry, and physics.
+``OceanBioME.jl`` can be used as a stand-alone box model, or integrated into ``Oceananigans.jl`` [@Oceananigans] simulations of ocean dynamics in one-, two-, or three-dimensions.
```

Comment:
```suggestion
``OceanBioME.jl`` can be used as a stand-alone box model, or integrated into ``Oceananigans.jl`` [@Oceananigans] simulations of ocean dynamics in one, two, or three dimensions.
```

I don't think hyphens are needed since here the numbers aren't being compound modifiers, right?
Hyphens would be appropriate if, e.g., the sentence read "in one-, two-, and three-dimensional problems".